src/NBitcoin/Networks.cs

```diff
@@ -266,12 +266,13 @@ private static Network InitReg()
         }
 
         private static Network InitStratisMain()
-        {            
+        {
             Block.BlockSignature = true;
             Transaction.TimeStamp = true;
 
             var consensus = new Consensus();
 
+            consensus.NetworkOptions = new NetworkOptions() { IsProofOfStake = true };
```

Comment:
Hmm now I am not sure about `NetworkOptions` being a property of `Consensus` arghhh...